# Robot's new target position
float64 x
float64 y